hotsos/defs/scenarios/openstack/nova/bugs/lp1761062.yaml

```diff
@@ -0,0 +1,17 @@
+checks:
+  has_1761062:
+    input:
+      path: 'var/log/nova/nova-compute.log'
+    expr: '([\d-]+) ([\d:]+).\d{3} .+DestinationDiskExists: The supplied disk path .+ already exists, it is expected not to exist'
```

Comment:
So I think to match others there should only be a single parens in the regex:

([\d-]+) [\d:]+\.\d{3}

Of course lp1967956.yaml looks just like this :-/ so we need to ask the oracle (Ed) since now I'm seeing a mix in neutron/bugs yamls as well.

Reply:
I'll try explain this. The answer to whether or not result groups to match timestamps are required lies in understanding how search constraints are applied. When you perform a search using the "search" property your search will be subject to constraints applied by the search engine (searchkit) and that does not require any groups at all since it will search for a date/time in each line before executing the search expression. You then have the option to apply extra constraints using [search.constraints](https://hotsos.readthedocs.io/en/latest/contrib/language_ref/property_ref/shared_properties.html#search) and this is where you need to ensure result groups - see https://github.com/canonical/hotsos/blob/61b7e1b93f168fde079045fc3d4b21fb0281b766/hotsos/core/ycheck/engine/properties/search.py#L204 but basically you need 1 group that matches the date and an optional second that matches the time. Hope that makes sense.